### ISSUE TITLE
Remove name from query and mutation options

### ISSUE THREAD

**Issue by jim-hill-r** [NavAbility/NavAbilitySDK.jl#34]
Diana 0.2.0 doesn't support anonymous calls, so this feature is included.

Upgrading to Diana 0.2.1 will fix it. Diana 0.2.1 requires HTTP 0.9 which conflicts with Neo4j 2.0.
Once we finish pulling DFG out and remove all deps on DFG and Caesar, we can do the upgrade as Neo4j won't be included anymore.

**Comment by dehann** [comment]
> Neo4j Diana conflicts

Just in case you are talking about name conflicts.  Note on Julia, difference between using `using` and `import`.
```julia
import Neo4j
import Diana

#the you still have to declare the module for functions, it avoids name conflicts
nsoc = Neo4j.connect(...)
dsoc = Diana.connect(...)
```

**Comment by jim-hill-r** [comment]
Name in this context is GraphQL operation name. Nothing to do with JuliaLang. 

This is now unblocked since neo4j is decoupled from SDK since NavAbility DFG has its own repo.

Just not a priority in anyway right now. Just kinda an annoyance.

**Comment by Affie** [comment]
Should we standardize queries to this in v0.6

**Comment by GearsAD** [comment]
That’ll happen as we bring the queries across to sdkCommonGQL, I’m not too
stressed about the GQL naming anyway, with the Deloitte driver we can even
use anonymous queries so this shouldn’t be a problem

**Comment by Affie** [comment]
Closing because we will use common GQL
